Business Rules/Automated Onboarding Workflow for New Employees/Script.js

```diff
@@ -0,0 +1,25 @@
+// Business Rule: Trigger onboarding process on new employee creation
+(function executeRule(current, previous /*null when async*/) {
+    if (current.active) {
+        // Create tasks for onboarding
+        var onboardingTasks = ['Set up email', 'Create user account', 'Assign equipment', 'Schedule orientation'];
+        
+        onboardingTasks.forEach(function(task) {
+            var taskGr = new GlideRecord('task');
+            taskGr.initialize();
+            taskGr.short_description = task;
+            taskGr.assigned_to = current.manager; // Assign to the manager
+            taskGr.state = 1; // Set to Open
+            taskGr.insert();
+        });
+
+        // Send notification to the manager
+        var email = new GlideEmailOutbound();
```

Comment:
Notifications should be triggered in the notification system rather than directly in the script.